app/utils/lava_log_parser.py

```diff
@@ -105,7 +155,11 @@ def run(log, boot, txt, html):
         "Boot result": boot_result_html,
         "Errors": numbers["error"],
         "Warnings": numbers["warning"],
+        "Kernel Panic": kernel_numbers["0"],
```

Comment:
How about having a sum of log levels 0~2?

Otherwise, alert and critical messages won't appear in this total.

Reply:
This should do the job:
```python
"Kernel Panic": sum(kernel_numbers[level] for level in "012"),
```

Reply:
Ah also, a kernel panic is something rather well defined in Linux.  Any messages at these log levels is not necessarily an actual panic.  So maybe having just "Kernel Errors" with the sum of levels 0-3 would actually be clearer, then warnings for level 4.